tests/theia_automation_lsp/inc/__init__.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 Broadcom.
```

Comment:
same here

Reply:
Files like __init__ are needed to be able to import modules from other files